### ISSUE TITLE
Submit a ticket

### ISSUE THREAD

**Issue by mgifford** [gctools-outilsgc/gcpedia#135]
Accessibility should be an optional category when submitting tickets here:
  https://wiki.gccollab.ca/Help_Aide

Furthermore, the list of accessibility tickets should be open to allow anyone to access them. It should be possible to see the list of known accessibility issues so that priorities can be identified and people can see progress is being made.
